pom.xml

```diff
@@ -121,9 +121,9 @@ flexible messaging model and an intuitive client API.</description>
     <testRealAWS>false</testRealAWS>
     <testRetryCount>1</testRetryCount>
 
-    <bookkeeper.version>4.7.0</bookkeeper.version>
+    <bookkeeper.version>4.8.0-SNAPSHOT</bookkeeper.version>
     <zookeeper.version>3.5.4-beta</zookeeper.version>
-    <netty.version>4.1.21.Final</netty.version>
+    <netty.version>4.1.22.Final</netty.version>
```

Comment:
We should try to stick with latest stable (4.1.25) since it has bugfixes, if there are no conflicts with grpc

Reply:
I hesitated to do it for 2.1.0 release, since I am not sure if the current grpc 1.12.0 works with 4.1.25.  I would suggest sticking to 4.1.22, we can look into bumping it into 4.1.25.